gempy/display/connection.py

```diff
@@ -86,7 +86,8 @@ def __new__(cls, viewer='ds9', path=None, use_existing=False,
                                    browser=browser)
 
         instance.logfile = 'imexam_log.txt'  # default logfile name
-        instance.log = imexam.util.set_logging()  # points to the package logger
+        if have_xpa:
```

Comment:
Why is the `imexam` logging only applicable for `ds9`? Is that just an `imexam` quirk?

Reply:
It's an imexam thing, but anyway our integration of ginga is done via imexam so that would be the same.